src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -36,13 +45,28 @@ public void run() {
 		}
 		try {
 			final TestReport report = TestReportUtil.load( testReport );
-			final TestReportPrinter printer = new TestReportPrinter( none(), loadRecheckIgnore() );
+			final TestReportPrinter printer = new TestReportPrinter( none(), checkFilterNames() );
 			logger.info( "\n{}", printer.toString( report ) );
 		} catch ( final IOException e ) {
 			logger.error( "Differences couldn't be printed:", e );
 		}
 	}
 
+	CompoundFilter checkFilterNames() throws IOException {
```

Comment:
Always handle exceptions where appropriate. `checkFilterNames` should catch the Exception and warn about it, not printing the report as discussed but instead providing a detailed message on why the report could not be printed.

Hint: "Differences couldn't be printed: exception" is not a good message since a user might not know what the exception means in context or where it originated. We should try not printing exceptions but instead provide a message on what failed.